packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -3332,7 +3332,7 @@ export type SignUpInWorkspaceMutationVariables = Exact<{
   workspaceInviteHash?: InputMaybe<Scalars['String']>;
   workspacePersonalInviteToken?: InputMaybe<Scalars['String']>;
   captchaToken?: InputMaybe<Scalars['String']>;
-  workspaceId: Scalars['UUID'];
+  workspaceId?: InputMaybe<Scalars['UUID']>;
```

Comment:
Let's stick to our single goal of this PR otherwise we might introduce breaking changes we don't notice